src/sidebar/services/load-annotations.js

```diff
@@ -16,53 +16,73 @@ export default function loadAnnotationsService(
   let searchClient = null;
 
   /**
-   * Load annotations for all URIs and groupId.
+   * Load all annotations that match `options` criteria
    *
-   * @param {string[]} uris
-   * @param {string} groupId
+   * @param {Object} options
+   *   @param {string} options.groupId
+   *   @param {string[]} [options.uris]
```

Comment:
I suggest to called it `urls`.

https://danielmiessler.com/study/difference-between-uri-url/#:~:text=A%20URI%20is%20an%20identifier,as%20HTTPs%20%2C%20FTP%20%2C%20etc.

Reply:
I'm trying to leave this particular one as I found it, as my refactoring here doesn't affect the way that `uris` (or `urls`) are handled—that is, your suggestion is reasonable, but it's outside of the scope of what I'm changing here.